src/main/java/com/google/api/codegen/transformer/ServiceTransformer.java

```diff
@@ -34,7 +36,20 @@ public ServiceDocView generateServiceDoc(
       docLines.add("");
       docLines.addAll(namer.getDocLines(conf.getManualDoc()));
     }
-    serviceDoc.lines(docLines.build());
+    List<String> lines = docLines.build();
```

Comment:
For encapsulation, would it make sense to keep `serviceDoc.lines(docLines.build())` here and move the logic below into `firstLine() and `remainingLines()`

Reply:
Oh, I see the recommendation in #1411 was to not do this. I would prefer to do it, but we don't need to re-litigate.

Reply:
I will leave as-is for now.